tests/test_D0_modelling_and_optimization.py

```diff
@@ -49,32 +70,24 @@ def test_if_model_building_time_measured_and_stored():
     assert isinstance(dict_values[SIMULATION_RESULTS]["modelling_time"], float)
 
 
-START_TIME = "2020-01-01 00:00"
-PERIODS = 3
-pandas_DatetimeIndex = pd.date_range(start=START_TIME, periods=PERIODS, freq="60min")
-
-dict_values_minimal = {
-    SIMULATION_SETTINGS: {TIME_INDEX: pandas_DatetimeIndex},
-    ENERGY_BUSSES: "bus",
-}
-
-
-def test_energysystem_initialized():
+def test_energysystem_initialized(dict_values_minimal):
     model, dict_model = D0.model_building.initialize(dict_values_minimal)
     for k in ("busses", "sinks", "sources", "transformers", "storages"):
         assert k in dict_model.keys()
     assert isinstance(model, oemof.solph.network.EnergySystem)
 
 
-def test_oemof_adding_assets_from_dict_values_passes():
+def test_oemof_adding_assets_from_dict_values_passes(dict_values):
```

Comment:
Interesting. I always thought that 

>def test_oemof_adding_assets_from_dict_values_passes(dict_values):

Why do you add it here while in other cases it is not necessary?
needed to have no argument, ie

>def test_oemof_adding_assets_from_dict_values_passes():

Reply:
It is a [pytest fixture](https://docs.pytest.org/en/stable/fixture.html), here `dict_values` correspond to the function `dict_values()` defined above